### PR TITLE
add tests for `tasks/observability/logs/telemetry-api`

### PR DESCRIPTION
**Pull request by zirain**
fixes: #12797

### CODE REVIEW

**Comment by zirain** [comment]
cc @lei-tang ready for review, PTAL.

File: content/en/docs/tasks/observability/logs/telemetry-api/test.sh [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+source "tests/util/samples.sh"
+
+# @setup profile=default
```
[the remark this answers is not in the document]
Comment (zirain) [reply content/en/docs/tasks/observability/logs/telemetry-api/test.sh:24]: there's too much test in none profile.

[the remark this answers is not in the document]
Comment (frankbu) [reply content/en/docs/tasks/observability/logs/telemetry-api/test.sh:24]: What do you mean "too much test in none profile"?

This test should definitely use `profile=none` and should be calling `snip_cleanup_3` at the end.

[the remark this answers is not in the document]
Comment (zirain) [reply content/en/docs/tasks/observability/logs/telemetry-api/test.sh]: my fault, I forgot that we run test on demand.

**Review by lei-tang — COMMENTED** [review]
LGTM. Thanks!

**Review by frankbu — APPROVED** [review]

**Pull request merged**